### PR TITLE
Improve nightly stability for ItIntrospectVersion.testUpdateImageName

### PR DESCRIPTION
**Pull request by maggiehe00**
Add  logic to make sure all the Services/Pods are ready after rolling start.

With this PR change the first 3 jobs on the KIND Jenkins passed in a row without any failure

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2379/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2380/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2381/

The second 3 jobs in a row
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2391/   -- all passed
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2392/ -- 1 failure
oracle.weblogic.kubernetes.ItMiiUpdateDomainConfig.testMiiUpdateWebLogicCredential
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2393/testReport/ -- no failure but 1 skipped. ItIntrospectVersion.testUpdateImageName passed though.

After addressing the review comment Jenkins job passed at:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/2419/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 1]
```diff
@@ -906,6 +906,26 @@ public void testUpdateImageName() {
     assertTrue(verifyRollingRestartOccurred(podsWithTimeStamps, 1, domainNamespace),
         String.format("Rolling restart failed for domain %s in namespace %s", domainUid, domainNamespace));
 
+    // verify the admin server service created
+    checkServiceExists(adminServerPodName, domainNamespace);
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:910]: Use the method checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace)
to combine  checkServiceExists() and checkPodReady()

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java [hunk 2]
```diff
@@ -906,6 +906,26 @@ public void testUpdateImageName() {
...
+      logger.info("Checking managed server service {0} is created in namespace {1}",
+          managedServerPodNamePrefix + i, domainNamespace);
+      checkServiceExists(managedServerPodNamePrefix + i, domainNamespace);
+    }
+
+    // verify cluster managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Waiting for managed server pod {0} to be ready in namespace {1}",
+          managedServerPodNamePrefix + i, domainNamespace);
+      checkPodReady(managedServerPodNamePrefix + i, domainUid, domainNamespace);
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java:926]: Use the method checkPodReadyAndServiceExists()  to combine checkServiceExists() and checkPodReady()

**Review by anpanigr — COMMENTED** [review]
Can u update the PR description, the reason for instability and how the proposed change will resolve the issue ?

**Review by sankarpn — APPROVED** [review]
As long as Pani's requests are fixed. I am good with the changes.

**Review by anpanigr — APPROVED** [review]
LGTM

**Pull request merged**